src/main/resources/application.yml

```diff
@@ -103,7 +103,32 @@ management:
 domain-events-sqs:
   provider: aws
   endpoint.region: eu-west-2
+  aws:
```

Comment:
Added this to allow the int tests to work - as the config beans cant be created without these values defined. However, these values will not be used in a deployed environment (like dev) as they are overridden by the actual values